frontend/packages/knative-plugin/src/components/overview/__tests__/overview-knative-test-data.ts

```diff
@@ -0,0 +1,21 @@
+import { RevisionModel } from '@console/knative-plugin/src/models';
+import { KnativeResourceOverviewPageProps } from '../KnativeResourceOverviewPage';
+
+export const mockOverviewItemData: KnativeResourceOverviewPageProps = {
+  item: {
+    buildConfigs: [],
+    obj: {
+      apiVersion: 'serving.knative.dev/v1beta1',
+      kind: 'Revision',
+      metadata: {
+        name: 'tag-portal-v1-sh6rp',
+        namespace: 'serverless-test',
+        uid: '2e4c5a8f-fa1e-4a75-b6dc-d67e4c8f8cc0',
+      },
+    },
+    routes: [],
+    services: [],
+  },
```

Comment:
can we get this i.e revision data from knative-topology data and avoid this file?

Reply:
revision data from knative-topology-test-data can be used only for `item.obj`. But `mockOverviewItemData.item` has to conform to the type `OverviewItem`. So we will need to create another object from it with the missing pieces of `OverviewItem`.

Reply:
@nemesis09 yes for `item.obj` you can pass revision data from knative-topology as other fields you have is anyway empty list so why to recreate `obj` again as we need to maintain. WDYT?

Reply:
We can do that, it will make one less file to maintain for sure.
I have updated the code accordingly, PTAL